### PR TITLE
Update SonarLint version, follow API changes

### PR DESCRIPTION
**Pull request by janos-ss**

### CODE REVIEW

File: src/main/java/org/sonarlint/cli/InputFileFinder.java [hunk 1]
```diff
@@ -153,5 +154,15 @@ public Charset getCharset() {
...
     }
+
+    @Override
+    public InputStream inputStream() throws IOException {
+      return Files.newInputStream(path);
+    }
+
+    @Override
+    public String contents() throws IOException {
+      return new String(Files.readAllBytes(path));
```
Comment (SonarTech) [inline src/main/java/org/sonarlint/cli/InputFileFinder.java]: MAJOR Remove this use of constructor "String(byte[])" [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1943)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by janos-ss** [comment]
Fixes https://jira.sonarsource.com/browse/SLCLI-23

**Comment by janos-ss** [comment]
Fixes https://jira.sonarsource.com/browse/SLCLI-24

**Pull request merged**